src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ManageContent/ManageContentPageViewModel.cs

```diff
@@ -87,7 +87,7 @@ public class PublicationViewModel
 
             public Contact Contact { get; set; }
 
-            public List<IdTitleViewModel> Methodologies { get; set; }
+            public List<Content.Services.ViewModels.MethodologyVersionSummaryViewModel> Methodologies { get; set; }
```

Comment:
Minor nitpick: just wondering if this could be shortened down to:

```cSharp
            public List<MethodologyVersionSummaryViewModel> Methodologies { get; set; }
```